### ISSUE TITLE
CT: multiple Errors for GrabDragInteraction

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/scenery-phet#602]
GrabDragInteraction is failing CT, see below.  I'm not sure who to assign this to, so I'll assign to @zepumph who has made the most recent commits.

```
scenery-phet : top-level-unit-tests : require.js
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/scenery-phet-tests.html
1 out of 2 tests passed. 1 failed.
GrabDragInteraction: GrabDragInteraction defaults failed:
ariaLabel should include thing string
at testDefaultDraggable (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteractionTests.js:62:12)
at Object.<anonymous> (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteractionTests.js:72:3)

GrabDragInteraction: GrabDragInteraction defaults failed:
aria role description should be same as model label
at testDefaultDraggable (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteractionTests.js:67:12)
at Object.<anonymous> (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteractionTests.js:72:3)


id: Bayes Chrome

scenery-phet : top-level-unit-tests : require.js?brand=phet-io
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/scenery-phet-tests.html?brand=phet-io
1 out of 2 tests passed. 1 failed.
GrabDragInteraction: GrabDragInteraction defaults failed:
Uncaught TypeError: Cannot read property 'dataStream' of undefined
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/tandem/js/PhetioObject.js:402

GrabDragInteraction: GrabDragInteraction defaults failed:
ariaLabel should include thing string
at testDefaultDraggable (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteractionTests.js:62:12)
at Object.<anonymous> (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteractionTests.js:72:3)

GrabDragInteraction: GrabDragInteraction defaults failed:
aria role description should be same as model label
at testDefaultDraggable (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteractionTests.js:67:12)
at Object.<anonymous> (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteractionTests.js:72:3)

GrabDragInteraction: GrabDragInteraction defaults failed:
Uncaught TypeError: Cannot read property 'dataStream' of undefined
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/tandem/js/PhetioObject.js:402

GrabDragInteraction: GrabDragInteraction defaults failed:
default to grabbable
at testDefaultGrabbable (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteractionTests.js:43:12)
at Object.<anonymous> (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteractionTests.js:76:3)

GrabDragInteraction: GrabDragInteraction defaults failed:
grabbable defaults to button
at testDefaultGrabbable (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteractionTests.js:44:12)
at Object.<anonymous> (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteractionTests.js:76:3)

GrabDragInteraction: GrabDragInteraction defaults failed:
no role for grabbable
at testDefaultGrabbable (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteractionTests.js:45:12)
at Object.<anonymous> (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteractionTests.js:76:3)

GrabDragInteraction: GrabDragInteraction defaults failed:
no aria-label for grabbable
at testDefaultGrabbable (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteractionTests.js:46:12)
at Object.<anonymous> (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteractionTests.js:76:3)

GrabDragInteraction: GrabDragInteraction defaults failed:
grabbable defaults to button html element.
at testDefaultGrabbable (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteractionTests.js:49:12)
at Object.<anonymous> (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteractionTests.js:76:3)


id: Bayes Chrome

scenery-phet : top-level-unit-tests : require.js?ea&brand=phet-io
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/scenery-phet-tests.html?ea&brand=phet-io
1 out of 2 tests passed. 1 failed.
GrabDragInteraction: GrabDragInteraction defaults failed:
Died on test #2 at https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteractionTests.js:20:7: Assertion failed: boolean expected
Error: Assertion failed: boolean expected
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/assert/js/assert.js:22:13)
at supportsGestureDescription (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteraction.js:59:13)
at new GrabDragInteraction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteraction.js:220:38)
at Object.<anonymous> (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588034137319/scenery-phet/js/accessibility/GrabDragInteractionTests.js:37:23)


id: Bayes Chrome
```

**Comment by samreid** [comment]
This has been happening for 6 days, can we please elevate the priority?  I'll also co-assign @jessegreenberg in case he knows more.

**Comment by jessegreenberg** [comment]
The unit tests identified in https://github.com/phetsims/scenery-phet/issues/602#issue-607961351 were fixed, what is showing up in CT now is 

```
scenery-phet : top-level-unit-tests : require.js?brand=phet-io
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588626350940/scenery-phet/scenery-phet-tests.html?brand=phet-io
1 out of 2 tests passed. 1 failed.
GrabDragInteraction: GrabDragInteraction defaults failed:
Uncaught TypeError: Cannot read property 'dataStream' of undefined
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588626350940/tandem/js/PhetioObject.js:398

GrabDragInteraction: GrabDragInteraction defaults failed:
Uncaught TypeError: Cannot read property 'dataStream' of undefined
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588626350940/tandem/js/PhetioObject.js:398

GrabDragInteraction: GrabDragInteraction defaults failed:
default to grabbable
at testDefaultGrabbable (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588626350940/scenery-phet/js/accessibility/GrabDragInteractionTests.js:47:12)
at Object.<anonymous> (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588626350940/scenery-phet/js/accessibility/GrabDragInteractionTests.js:80:3)

GrabDragInteraction: GrabDragInteraction defaults failed:
grabbable defaults to button
at testDefaultGrabbable (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588626350940/scenery-phet/js/accessibility/GrabDragInteractionTests.js:48:12)
at Object.<anonymous> (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588626350940/scenery-phet/js/accessibility/GrabDragInteractionTests.js:80:3)

GrabDragInteraction: GrabDragInteraction defaults failed:
no role for grabbable
at testDefaultGrabbable (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588626350940/scenery-phet/js/accessibility/GrabDragInteractionTests.js:49:12)
at Object.<anonymous> (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588626350940/scenery-phet/js/accessibility/GrabDragInteractionTests.js:80:3)

GrabDragInteraction: GrabDragInteraction defaults failed:
no aria-label for grabbable
at testDefaultGrabbable (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588626350940/scenery-phet/js/accessibility/GrabDragInteractionTests.js:50:12)
at Object.<anonymous> (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588626350940/scenery-phet/js/accessibility/GrabDragInteractionTests.js:80:3)

GrabDragInteraction: GrabDragInteraction defaults failed:
grabbable defaults to button html element.
at testDefaultGrabbable (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588626350940/scenery-phet/js/accessibility/GrabDragInteractionTests.js:53:12)
at Object.<anonymous> (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1588626350940/scenery-phet/js/accessibility/GrabDragInteractionTests.js:80:3)
```

But when I run scenery-phet unit tests locally, all tests are passing.

**Comment by jessegreenberg** [comment]
phet.phetio.dataStream is undefined in unit tests.

**Comment by jessegreenberg** [comment]
Would be helpful to talk about during tomorrows a11y-dev meeting.

Can we run scenery-phet unit tests without ?phet-io? Should we reconsider GrabDragInteraction and its unit tests? GrabDragInteraction requires a sim (for utteranceQueue) and there are stubs at the top of the file so that it works outside of a sim context but that doesn't seem very robust.

**Comment by samreid** [comment]
I switched to use `qunitStart()` and it seems to be passing phet-io brand now.  OK to close?

**Comment by jessegreenberg** [comment]
Cool, its working great, thanks! Closing.
